services/provider/server/server.go

```diff
@@ -407,9 +407,9 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 				},
 			},
 			Quota: corev1.ResourceQuotaSpec{
-				Hard: corev1.ResourceList{"requests.storage": *resource.NewScaledQuantity(
-					int64(consumerResource.Spec.StorageQuotaInGiB),
-					resource.Giga,
+				Hard: corev1.ResourceList{"requests.storage": *resource.NewQuantity(
+					int64(consumerResource.Spec.StorageQuotaInGiB*1024*1024*1024),
```

Comment:
reading only this line seems a bit confusing, did it occur to you that if only this line is read, it feels like we are multiplying a value which has GiB in it's name by 1024^3 resulting in something like Exbibyte?

maybe it only occurred to me, since the conversion is correct, approving it.

Reply:
resolve.